### PR TITLE
using istioctl

### PR DESCRIPTION
**Pull request by tt-le**
Updated deploy istio section to use `istioctl` for installation and update tearing down section with `istioctl`.
Related to Issue:  @[OpenLiberty/guide-istio-intro](https://github.com/OpenLiberty/guide-istio-intro/issues/106)

### CODE REVIEW

**Review by gkwan-ibm — COMMENTED** [review]
@tt-le  at "Enabling MicroProfile Fault Tolerance" section
- change "Navigate to the `start` directory..." -> "Navigate to the `guide-{projectid}\start` directory..."

File: README.adoc [hunk 1]
```diff
@@ -105,91 +105,25 @@ include::{common-includes}/gitclone.adoc[]
...
+include::{common-includes}/istio-start.adoc[]
 
 // =================================================================================================
 // Enabling MicroProfile Fault Tolerance
 // =================================================================================================
 
 == Enabling MicroProfile Fault Tolerance
 
-Navigate to the `start` directory to begin.
+Navigate to the `guide-{projectid}\start` directory to begin.
```
Comment (gkwan-ibm) [inline README.adoc:126]: @tt-le  change `guide-{projectid}\start` to `guide-{projectid}/start`

File: README.adoc [hunk 2]
```diff
@@ -246,7 +180,7 @@ The integer value must be greater than or equal to -1. A value of -1 indicates t
 
 === Building and running the application
 
-Navigate to the `start` directory and run the following command to build your application and integrate the Retry policy into your microservices:
+Navigate to the `guide-{projectid}\start` directory and run the following command to build your application and integrate the Retry policy into your microservices:
```
Comment (gkwan-ibm) [inline README.adoc:183]: @tt-le change `guide-{projectid}\start` to `guide-{projectid}/start`

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]

**Comment by tt-le** [comment]
Addressed feedback. Changed the README to use the guides-commons adoc.

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**